mlflow-v2-examples/kserve-example.ipynb

```diff
@@ -0,0 +1,440 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "b46a8ea0-e797-4e09-8b1a-7fd6036e48cc",
+   "metadata": {},
+   "source": [
+    "# MLflow K-serve example integration\n",
+    "This example shows how to train a regressor with scikit. Store the model object in the MLflow and deploy the same object with the K-serve. This example expects the Kubeflow and the MLflow bundle being deployed. Please reffer to [this guide](https://documentation.ubuntu.com/charmed-mlflow/en/latest/tutorial/mlflow-kubeflow/). Following environment variables are expected to be set (in case of charmed kubeflow they are set for you):\n",
+    "\n",
+    "* MLFLOW_S3_ENDPOINT_URL: endpoint for object storage \n",
+    "* MLFLOW_TRACKING_URI: endpoint for mlflow server\n",
+    "* AWS_SECRET_ACCESS_KEY: secret key for object storage\n",
+    "* AWS_ACCESS_KEY_ID: username for object storage"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "d6bf6636-f4cf-4c0e-88e5-e3539af29cdb",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "AWS_ACCESS_KEY_ID=minio\n",
+      "AWS_SECRET_ACCESS_KEY=08MND4UU8TAW48U0TIZMN10N994YNU\n"
+     ]
+    }
+   ],
+   "source": [
+    "!printenv | grep AWS"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "96621cb1-da6c-4a2e-94b6-bad7b8f1dd9a",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "MLFLOW_TRACKING_URI=http://10.1.137.14:5000/\n",
+      "MLFLOW_S3_ENDPOINT_URL=http://10.1.137.21:9000/\n"
+     ]
+    }
+   ],
+   "source": [
+    "!printenv | grep MLFLOW"
+   ]
+  },
```

Comment:
Regarding this:
- I think we should split the first two sentences in something like (along with some rephrasing):
```
This example shows how to train a regressor with scikit. It stores the model object in MLflow and deploys the same object using KServe.
```
and then have a separate section
### Prerequisites
with
```
This example expects the Kubeflow and the MLflow bundle being deployed. Please refer to [this guide](https://documentation.ubuntu.com/charmed-mlflow/en/latest/tutorial/mlflow-kubeflow/). Following environment variables are expected to be set (in case of charmed kubeflow they are set for you):
...
!printenv | grep MLFLOW
MLFLOW_TRACKING_URI=http://10.1.137.14:5000/
MLFLOW_S3_ENDPOINT_URL=http://10.1.137.21:9000/
```
- typo in refer (in "Please reffer to this guide")